templates/repo/commits_list_group_by_date.tmpl

```diff
@@ -0,0 +1,111 @@
+			{{range $groupCommit.Commits}}
+			{{$commitRepoLink := $.RepoLink}}{{if $.CommitRepoLink}}{{$commitRepoLink = $.CommitRepoLink}}{{end}}
+				<li class="commits-list-item">
+					<div class="tw-pt-4 tw-pl-4 title">
+						<h4>
+							<span class="message-wrapper">
+							{{if $.PageIsWiki}}
+								<span class="commit-summary {{if gt .ParentCount 1}} grey text{{end}}" title="{{.Summary}}">{{.Summary | ctx.RenderUtils.RenderEmoji}}</span>
+							{{else}}
+								{{$commitLink:= printf "%s/commit/%s" $commitRepoLink (PathEscape .ID.String)}}
+								<span class="commit-summary {{if gt .ParentCount 1}} grey text{{end}}" title="{{.Summary}}">{{ctx.RenderUtils.RenderCommitMessageLinkSubject .Message $commitLink ($.Repository.ComposeCommentMetas ctx)}}</span>
+							{{end}}
+							</span>
+							{{if IsMultilineCommitMessage .Message}}
+								<button class="ui button ellipsis-button" aria-expanded="false" data-global-click="onRepoEllipsisButtonClick">...</button>
+							{{end}}
+							{{if IsMultilineCommitMessage .Message}}
+								<pre class="commit-body tw-hidden">{{ctx.RenderUtils.RenderCommitBody .Message ($.Repository.ComposeCommentMetas ctx)}}</pre>
+							{{end}}
+							{{if $.CommitsTagsMap}}
+								{{range (index $.CommitsTagsMap .ID.String)}}
+									{{- template "repo/tag/name" dict "RepoLink" $.Repository.Link "TagName" .TagName "IsRelease" (not .IsTag) -}}
+								{{end}}
+							{{end}}
+						</h4>
+					</div>
+					<div class="tw-flex tw-items-center tw-gap-1 tw-pb-4 tw-pl-4 description">
+						<div class="author">
+							{{$userName := .Author.Name}}
+							{{if .User}}
+								{{if and .User.FullName DefaultShowFullName}}
+									{{$userName = .User.FullName}}
+								{{end}}
+								{{ctx.AvatarUtils.Avatar .User 28 "tw-mr-2"}}<a class="muted author-wrapper" href="{{.User.HomeLink}}">{{$userName}}</a>
+							{{else}}
+								{{ctx.AvatarUtils.AvatarByEmail .Author.Email .Author.Name 28 "tw-mr-2"}}
+								<span class="author-wrapper">{{$userName}}</span>
+							{{end}}
+						</div>
+						<span>
+							{{if .Committer}}
+								committed
+							{{else}}
+								authored
+							{{end}}
+						</span>
+						{{if .Committer}}
+							{{DateUtils.TimeSince .Committer.When}}
+						{{else}}
+							{{DateUtils.TimeSince .Author.When}}
+						{{end}}
+						{{if .Statuses}}
+						<span>·</span>
+						{{end}}
+						{{template "repo/commit_statuses" dict "Status" .Status "Statuses" .Statuses}}
+					</div>
+					<div class="tw-flex tw-flex-wrap tw-items-center tw-gap-2 tw-pr-4 metadata">
+						{{$commitBaseLink := ""}}
+						{{if $.PageIsWiki}}
+							{{$commitBaseLink = printf "%s/wiki/commit" $commitRepoLink}}
+						{{else if $.PageIsPullCommits}}
+							{{$commitBaseLink = printf "%s/pulls/%d/commits" $commitRepoLink $.Issue.Index}}
+						{{else if $.Reponame}}
+							{{$commitBaseLink = printf "%s/commit" $commitRepoLink}}
+						{{end}}
+						<div class="commit_sign_badge">
```

Comment:
just a nit may forgatten

```suggestion
						<div class="commit-sign-badge">
```

Reply:
Actually it is a wrong class ......

The real `class="commit-sign-badge"` is only used in `commit_sign_badge.tmpl`